include/sound/sof/dai-intel.h

```diff
@@ -173,7 +173,7 @@ struct sof_ipc_dai_dmic_params {
 	uint32_t fifo_fs;	/**< FIFO sample rate in Hz (8000..96000) */
 	uint32_t reserved_1;	/**< Reserved */
 	uint16_t fifo_bits;	/**< FIFO word length (16 or 32) */
-	uint16_t reserved_2;	/**< Reserved */
+	uint16_t fifo_bits_b;	/**< Deprecated since 3.0.1 */
```

Comment:
is 3.0.1 the ABI level? maybe clarify is this is firmware version or ABI to remove any ambiguity.

Reply:
Yep. It's for loaded firmware ABI level that is got via sdev->fw_ready->version.

Reply:
Done